### ISSUE TITLE
Fortnox paging. 

### ISSUE THREAD

**Issue by MagnusVMI** [wetcat-studios/fortie#32]
Hi,
I have some issues that I only got 100 rows back from fortnox.
Because there paging the response.
Have I missed some settings, or do I have to make the call multiply times?

Thank you for a great job Btw...

/ Magnus

**Comment by agoransson** [comment]
I will research this and get back to you this week.

**Comment by MagnusVMI** [comment]
Okey. 
Just say if it is something I can do/help you with.

**Comment by agoransson** [comment]
@MagnusVMI there's a fix in version `1.0.1` that should work for you. I haven't tested all providers though, but I assumed that the implementation on Fortnox' side was identical for all routes so everything should work.

Providers that already had an optional parameter in the `all()` method has the `page` as second parameter.

Examples:
```
$fortie->accounts()->all(5); // Will retrieve page 5 of accounts, if it exists
$fortie->articles()->all(); // Will retrieve page 1 of articles, if it exists
$fortie->invoices()->all(null, 4); // Will retrieve page 4 of invoices, and apply no filter
```
